capstone/src/main/webapp/statistics_script.js

```diff
@@ -0,0 +1,221 @@
+async function getSurveyResponses(){
+    const response = await fetch('/load-data');
+    const list = await response.json(); //list of entities from datastore
+    let scores = new Array();
+    let sentimentCount = [0,0,0,0,0]; // Very positive, positive, neutral, negative, and very negative
+    const refGender = ['M','F','U'];
+    let genderCount = [0,0,0]; // Male, female, and unknown
+    const refAge = ['18-24','25-34','35-44','45-54','55-64','65+','Unknown'];
+    let ageCount = [0,0,0,0,0,0,0]; // 18-24, 25-34, 35-44, 45-54, 55-64, 65+, or Unknown
```

Comment:
Instead of using two arrays to track the counts for each, consider using a JSON object.
```
const refAge = {
  '18-24': 0,
...
}
```

Same goes for all the others. It would be more readable and easier to track than using indices.